### PR TITLE
Rename file to assure test be running first

### PR DESCRIPTION
**Pull request by ealanisg**

### CODE REVIEW

File: .structignore [hunk 1]
```diff
@@ -5,5 +5,5 @@ test/projects/assorted/@sls-api/test/errors.spec.js
 test/projects/assorted/@sls-api/test/routes.spec.js
 test/projects/assorted/@sls-api/test/handler.spec.js
 test/projects/assorted/@sls-queue/test/queue/queue.spec.js
-test/projects/assorted/@sls-closedsource/test/sls-cf-stack.spec.js
-test/projects/assorted/@sls-closedsource-react/test/sls-cf-stack.spec.js
+test/projects/assorted/@sls-closedsource/test/0_sls-cf-stack.spec.js
+test/projects/assorted/@sls-closedsource-react/test/0_sls-cf-stack.spec.js
```
Comment (ealanisg) [inline .structignore:9]: Renamed this too preventing in frontend they would have the same problem...

**Comment by simlu** [comment]
Please use `0-` (zero dash) not `0_` (zero underscore) as discussed

File: .structignore (hunk 1 above)
Comment (simlu) [reply .structignore:9]: There is a frontend? Seriously though we have no idea what you are talking about :)

Comment (ealanisg) [reply .structignore:9]: As I saw `react` here I supposed that this plugin was used by `frontend` team too, that's why the note...

**Pull request merged**